src/object.d

```diff
@@ -3995,39 +3995,33 @@ void __ArrayDtor(T)(T[] a)
         e.__xdtor();
 }
 
-/// See $(REF _d_cmain, core,internal,entrypoint)
+/**************************************************************************
```

Comment:
```suggestion
/* ************************************************************************
```

Otherwise it's a documentation comment

Reply:
I thought the newline between the comment and the declaration would handle that.